src/app/menu/menu.component.scss

```diff
@@ -93,4 +92,63 @@
 {
     padding-left:50px;
     transition: padding-left 0.5s;
+}
+
+@media(max-width: 700px){
```

Comment:
This we have to change 700px to the correct number.
Take a look: https://getbootstrap.com/docs/3.3/css/
Maybe this can help

Reply:
Will do it together